tests/e2e/standalone/run_template_test.go

```diff
@@ -0,0 +1,365 @@
+//go:build e2e || template
+// +build e2e template
+
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package standalone_test
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type AppTestOutput struct {
+	appID              string
+	appLogContents     []string
+	daprdLogContent    []string
+	baseLogDirPath     string
+	appLogDoesNotExist bool
+}
+
+func TestRunWithTemplateFile(t *testing.T) {
```

Comment:
@mukundansundar  Do we need to add one more tests where it picks the `dapr.yaml` from a given directory ? May be we can reuse the require and asserts below..

Reply:
I will add it.